app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -646,7 +646,7 @@ public void run() {
                 if (mVideoManager != null)
                     mVideoManager.start();
             }
-        }, 750);
+        },  userPreferences.getValue().get(UserPreferences.Companion.getVideoStartDelay()));
```

Comment:
The comment above (about why this is delayed) is inaccurate and could be removed. Additionally a fast-path when there is no delay would be nice to have.

Reply:
By fast-path I assumed you meant to add a condition to skip the mHandler.postDelayed logic if the delay is set to 0. I have added a check for this.

Reply:
Yup that's what I meant!